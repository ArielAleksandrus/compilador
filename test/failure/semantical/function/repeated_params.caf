int func1(int a, int a){
    retorne 0;
}

programa{}
