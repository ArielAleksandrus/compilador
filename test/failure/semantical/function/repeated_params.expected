ERROR => Line 1, col 22: Redeclaration of parameter. 'a' previously found in line 1, col 15
